### DISCUSSION TITLE
A message from Yamer

### DISCUSSION THREAD

**Discussion by YamerGrimlock** [InstantID/InstantID#90] [General]
Thank you for mentioning the use of Unstable Diffuser V8 Yamer MIX;, I am the creator of that checkpoint for SDXL 1.0.

Feel free to dm me on discord if you guys have plans to use my checkpoints for future projects: yamer_ai

To verify the authenticity of this profile please check my Civitai.com page ( https://civitai.com/user/Yamer ) and read a description of a model, I always write in the description my discord name or contact me on Twitter: @YamerOfficial.

I'm happy to see cool projects made using my diffuser checkpoints, you guys are fantastic and deserve the best. <3

**Comment by haofanwang** [comment]
Thank you for your great work! Follow you on Twitter now ❤️, I have added you in [acknowledgements](https://github.com/InstantID/InstantID/blob/main/README.md#acknowledgements).
